From 3b8e37f311b787e4f8c45600fcadaef41a55af07 Mon Sep 17 00:00:00 2001
From: "Barry Lind (SteeManMI)" <barrylind@yahoo.com>
Date: Tue, 29 Nov 2022 09:15:24 -0500
Subject: [PATCH] Fix the builddeb script so linux-image upgrades work
 correctly on fat /boot partitions

 Changes to be committed:
	modified:   packages/armbian/builddeb
---
 packages/armbian/builddeb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/packages/armbian/builddeb b/packages/armbian/builddeb
index 0163d401a..4fb13b53a 100755
--- a/packages/armbian/builddeb
+++ b/packages/armbian/builddeb
@@ -317,7 +317,7 @@ done
 ##
 sed -e "s/exit 0//g" -i $tmpdir/DEBIAN/postinst
 cat >> $tmpdir/DEBIAN/postinst <<- EOT
-	ln -sf $(basename $installed_image_path) /boot/$image_name 2> /dev/null || mv /$installed_image_path /boot/$image_name
+	ln -sf $(basename $installed_image_path) /boot/$image_name 2> /dev/null || cp /$installed_image_path /boot/$image_name
 	touch /boot/.next
 	exit 0
 EOT
-- 
2.34.1

